mathext/ell_carlson_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import (
+	"math"
+	"math/rand"
+	"testing"
+)
+
+// Testing EllipticF (and CarlsonRF) using the addition theorems from http://dlmf.nist.gov/19.11.i
+func TestEllipticF(t *testing.T) {
+	const tol = 1.0e-14
+	rng := rand.New(rand.NewSource(271829))
```

Comment:
Why is this seed chosen? If there is a reason, please comment. We normally seed 1 since that is the global initial seed.